client/src/components/LoginForm/index.js

```diff
@@ -1,23 +1,18 @@
-import React, { useCallback } from 'react';
+import React, { useState } from 'react';
 import Button from '@material-ui/core/Button';
 
+import Input from '../Input';
 import { makeRequest } from '../../utils/makeRequest';
-import useInput from '../../hooks/useInput';
 import useApp from '../../hooks/useApp';
 
 import './styles.scss';
 
 const LoginFrom = () => {
-  const [login, loginInput] = useInput({ label: 'Login', className: 'login-form__input', id: 'login-input' });
-  const [password, passwordInput] = useInput({
-    label: 'Password',
-    className: 'login-form__input',
-    type: 'password',
-    id: 'password-input',
-  });
+  const [login, setLogin] = useState('');
+  const [password, setPassword] = useState('');
```

Comment:
Previous time, didn't you mean something like that? https://rangle.io/blog/simplifying-controlled-inputs-with-hooks/

Reply:
@Xoxol yeah it's same way